### PR TITLE
Issue/887

### PR DESCRIPTION
**Pull request by danahertzberg**

### CODE REVIEW

File: site/content/docs/5.0/components/navbar-off-canvas.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Adding the `.navbar-offcanvas` class to your `<nav>` will incorporate an off-can
       </div>
     </form>
   </div>
-  <ul class="navbar-nav flex-lg-row">
+  <ul class="navbar-nav flex-lg-row pb-5">
```
Comment (danahertzberg) [inline site/content/docs/5.0/components/navbar-off-canvas.md:45]:
```suggestion
  <ul class="navbar-nav flex-lg-row">
```

**Comment by danahertzberg** [comment]
Look into stylelint config for Bootstrap (or Twitter bootstrap) to fix linting errors

**Comment by danahertzberg** [comment]
https://markdotto.com/2011/11/29/css-property-order/

Account for position-absolute potential issues. Connect with @trackleft

**Comment by trackleft** [comment]
I think we could set a css property via the javascript that affects the body overflow.

**Comment by joeparsons** [comment]
This is superseded by #899

**Pull request closed without merging**